c2cgeoform/scaffolds/c2cgeoform/package.json_tmpl

```diff
@@ -12,7 +12,8 @@
     "jquery": "^3.2.1",
     "jquery.scrollintoview": "1.9.4",
     "jquery-steps": "^1.1.0",
-    "openlayers": "=3.11.2",
-    "typeahead.js": "=0.10.5"
+    "openlayers": "4.5.0",
+    "typeahead.js": "=0.10.5",
+    "@camptocamp/closure-util": "1.23.0"
```

Comment:
Is there a reason to not use the closure-util version pointed in openlayers dependencies ?
On my system, everythings seems to work perfectly without pinning closure-util version here.

Reply:
Hmmm... I (on Windows) always had some problems with this version, but it does not really matter anymore with the newer ones. So in my opinion, we should not waste too much time on this and close this PR.

OK for you, @arnaud-morvan ?